README.md

```diff
@@ -107,6 +108,10 @@ And additional requirements are in [**requirements.txt**](https://github.com/gar
   * Migrate the changes to the database `$ python manage.py migrate`
   * Create admin `$ python manage.py createsuperuser`
   * Run the server `$ python manage.py runserver`
+  
+### 💡️**Pro Tip:** 
```

Comment:
"Pro Tip" looks too big. 4 `#` should do fine. Also, try putting a space between light emoji and Pro.